glide.lock

```diff
@@ -89,7 +89,7 @@ imports:
 - name: github.com/pkg/errors
   version: a22138067af1c4942683050411a841ade67fe1eb
 - name: github.com/prometheus/client_golang
-  version: faf4ec335fe01ae5a6a0eaa34a5a9333bfbd1a30
+  version: f504d69affe11ec1ccb2e5948127f86878c9fd57
```

Comment:
this version should have been locked https://github.com/arquillian/ike-prow-plugins/blob/master/glide.lock#L92